chapters/en/module3.md

```diff
@@ -12,19 +12,708 @@ id: 3
+Genre</text></g></g><path class="foreground" aria-hidden="true" d="" pointer-events="none" display="none"/></g></g></g><path class="foreground" aria-hidden="true" d="" display="none"/></g></g></g></svg>
+
+<choice id="1" >
+<opt text="There are too many records.">
+
+Having a lot of observations has no impact on a density plot. 
+
+</opt>
+
+<opt text="The majority of the observations lie to the left of the axis making the values and genres difficult to observe." correct="true">
+
+Got it! 
+
+</opt>
+
+<opt text="Their are too many genres to compare." >
+
+Too many genres shouldn't cause a problem for observing insights.
+
+</opt>
+
+
+<opt text="The x-axis tend to display too many values without reason." >
+
+If our plot includes the axis to the length shown, there are records for the values but they are so close to 0, that they cannot be identified. 
+</opt>
+
+</choice>
+
+
+**Question 2**      
+Which of the following statistics is **NOT** displayed in a boxplot?
+
+
+
+<choice id="2" >
+<opt text="Mean" correct="true">
+
+You've got the right argument but the wrong value being set. 
+
+</opt>
+
+<opt text="Maximum">
+
+Nice!
+
+</opt>
+
+<opt text="Minimum">
+
+I think your argument should be called something else.
+
+</opt>
+
+
+<opt text="Median">
+
+You are extremely close, but no past tense is needed.
+
+</opt>
+
+</choice>
+
+
+</exercise>
+
+<exercise id="13" title="Boxplots and Penguins">
+
+
+**Instructions:**    
+Be patient when running a coding exercise for the first time, it can take a few minutes. 
+
+**When you see `____` in a coding exercise, replace it with what you assume to be the correct code. Run the code to see if you obtain the desired output
+and submit it to validate if you were correct.**
+
+
+In the last few slides, we were faceting and using colour channels to compare the body mass of different species of penguins. Let's explore this but this time using a boxplot!
+
+<codeblock id="penguins">
+
+</codeblock>
+
+
+Tasks: 
+
+Fill in the blanks in the code below so that the following gets accomplished.
+
+- Use the data source `penguins` to make **boxplots** for the body mass values among the different penguin species.
+- Make sure to give it an appropriate title and set the plot dimensions to a height of 200 and a width of 400.
+- Remember to assign the `species` to the y-axis and the `body_mass_g` to the y-axis since we want to read the labels easily as we learned in the previous module.
+
+
+<codeblock id="03_13">
+
+- Are you using `mark_boxplot()`?
+- Are you setting `alt.X('body_mass_g')`?
+- Are you setting `alt.Y('species')`?
+- Are you setting a title, height and width in `properties()`?
+
+</codeblock>
+
+
+**Question**      
+
+Which species of penguin tend to have the most outliers?
```

Comment:
I really like this question! It would be good to have 2-3 more after this asking them to identify the highest median, the lowest minimum, and the highest maximum (or other parts of the box plot chart).

Reply:
Added!